aws/policy/compute.yaml

```
@@ -155,6 +155,7 @@ Statement:
       - autoscaling:TerminateInstanceInAutoScalingGroup
       - ec2:DeleteVolume
       - elasticloadbalancing:AddTags
+      - elasticloadbalancing:AddListenerCertificates
```

To keep the alphabetical ordering:
```suggestion
      - elasticloadbalancing:AddListenerCertificates
      - elasticloadbalancing:AddTags
```